sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_responses.py

```diff
@@ -0,0 +1,38 @@
+# The MIT License (MIT)
+# Copyright (c) 2024 Microsoft Corporation
+
+from azure.core.utils import CaseInsensitiveDict
+
+
+class CosmosDictResponse(dict):
+    def __init__(self, original_dict, response_headers):
```

Comment:
I think I would make the first param positional-only (also type hints):
```suggestion
    def __init__(self, original_dict: Mapping[str, Any], /, *, response_headers: Mapping[str, Any]) -> None:
```

Reply:
why would we not keep it for both if this is only meant to be used internally and we will always have both included? there's no scenario in which we do not include the response headers in the response